docs/docs/Home.md

```diff
@@ -28,7 +36,6 @@ A primary goal of AnitaB.org is to be inclusive of the largest number of contrib
 
 See the full [Code of Conduct](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CODE_OF_CONDUCT.md) and the [Reporting Guidelines](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/REPORTING_GUIDELINES.md).
 
-
 # Contact
 
-You can reach us at [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech). This project is discussed on the [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) stream.
\ No newline at end of file
+You can reach us at [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech). This project is discussed on the [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) stream.
```

Comment:
Why this is marked as changed? Please revert any white-space/tab changes here if done